src/main/java/org/folio/innreach/controller/InnReachLocationController.java

```diff
@@ -0,0 +1,75 @@
+package org.folio.innreach.controller;
+
+import java.util.UUID;
+
+import javax.validation.Valid;
+import javax.validation.constraints.Max;
+import javax.validation.constraints.Min;
+import javax.validation.constraints.Pattern;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import org.folio.innreach.domain.service.InnReachLocationService;
+import org.folio.innreach.dto.InnReachLocationDTO;
+import org.folio.innreach.dto.InnReachLocationsDTO;
+import org.folio.innreach.rest.resource.LocationsApi;
+
+@RequiredArgsConstructor
+@RestController
+@RequestMapping("/locations")
```

Comment:
Please pay attention here on the path location. As we mentioned in module descriptor template we uses modules nama space.

Reply:
Done